### ISSUE TITLE
stop complaining on deleted imports

### ISSUE THREAD

**Issue by oconnor663** [buildinspace/peru#23]
We want `peru sync` to be careful about overwriting the user's files. Peru doesn't pave over preexisting files, or any changes that have been made since a file was created. This is to avoid accidentally deleting users' work, and also to try to catch some of the cases where users have done the Wrong Thing with peru (like checking in synced files (#dowhatisaynotwhatido)). But currently we also freak out if the user has _deleted_ files that peru synced, and I think that might be overzealous. Consider this scenario:
1. I want to clean junk out of my repo, but I don't want to lose my peru cache.
2. So I run `git clean -dfx --exclude .peru`. Maybe I have an alias for that.
3. Now I call `peru sync`. Peru absolutely refuses until I use `-f`.

I think it would be better if peru stopped complaining here. There's no risk of losing work, and it's not really catching any Wrong Things. If a user sees this error all the time, they're not going to be paying attention when it eventually catches a real mistake. (Especially if we get them in the habit of using `-f`.)

tldr: `peru sync` should consider deleted files "clean".
